lib/active_merchant/billing/gateways/global_collect.rb

```diff
@@ -114,7 +118,7 @@ def add_order(post, money, options, capture: false)
         post['order']['references']['invoiceData'] = {
           'invoiceNumber' => options[:invoice]
         }
-        add_airline_data(post, options)
+        add_airline_data(post, options) unless ogone_direct?
```

Comment:
Why is `airline_data` only available for ogone_direct?

Reply:
No, it's the opposite, `airline_data` is only for `GlobalCollect` if `ogone_direct` is false we fill this information.